README.md

```diff
@@ -59,48 +150,30 @@ or
 source ~/.zshrc
 ```
 
----
-
-## Setup for Local Virtual Machines
-
-On the DeEEP Device - the main Ansible Controller is the Device itself which in turn can manage the VM's (via Incus). Each VM is also an independent Ansible Controller for itself. This setup requires every role to be `localhost` first when developed. This isn't the normal way Ansible is meant to be used (push model) so there are some gotchas and things we do that are slightly out of the norm. Being `localhost` first however provides the VMs the ability to configure themselves when necessary. It's also important to note that if it works via `localhost` it will always work being pushed (not always the case in reverse).
 
-> [!NOTE] For testing, we skip the Incus layer. Meaning the Ansible Controller for tests is your local machine (acting like the Device). Pushing directly to the VM running locally (managed by Orbstack, multipass, KVM, or LXD). This is meant to simplify the testing process. Especially since MacOS support for intel based CPU architecture is not well supported.
 
-As such, depending on your development environment pick the most convenient virtualization software. Usually based on OS and CPU
-
----
+# Getting Started
 
-### macos (apple silicon)
+> **Important**: Before proceeding, ensure you have set your `HAB_AUTH_TOKEN` environment variable. This token is required for authenticating with the Habitat Builder service and is expected to be set before continuing.
+>
+> ```bash
+> export HAB_AUTH_TOKEN='your-token-here'
+> ```
 
-#### orbstack - [docs](https://docs.orbstack.dev/install)
+From the root of the bedrock repo run the following command:
 
 ```bash
-brew install orbstack
+molecule create
 ```
 
----
-
```

Comment:
Might want to include how to access the VM via orb is applicable or linking to their documents etc if it makes sense from a newbie perspective?